### ISSUE TITLE
question

### ISSUE THREAD

**Issue by amongussusususu** [ShadiestGoat/jklm-bombparty-api#1]
is it possible to get all the jklm dictionaries? just asking

**Comment by ShadiestGoat** [comment]
No. I'm working on this API right now, but JKLM never exposes it's stuff. I will be making a bot soon:tm: that will hopefully fetch such stuff, but nothing as of now. 

Once this wrapper is more stable & has propper docs, the idea would be to go into a run w/ a bot, and spam either random letters, or from a dictionary. Whenever one is accepted, add it to a db. Also add any words from the bot. 

But again, as of now, its impossible
